components/ResultsContainer.vue

```diff
@@ -11,7 +11,14 @@
     >
       <div class="d-flex align-items-center">
         <h3
-          v-if="Object.is(results, null)"
+          v-if="!Object.is(error, null)"
```

Comment:
```suggestion
          v-if="error !== null"
```
Any reason not to just do this? If I understand this correctly, the only reason to use `Object.is` here is if you would like to use `NaN` as a signal for "no error", because that'd behave differently in `===`. But I don't think you're using `NaN` and if you did, we should probably find another signal for "no error" anyway.

I'd prefer `===` just because more commonly used here, i.e. more expected and readable. LMK if there is a strong reason not to do that.

Reply:
There isn't a strong reason not to do that, just multiple ways of doing the same thing.